.github/workflows/dart.yml

```diff
@@ -36,12 +36,12 @@ jobs:
       - uses: actions/checkout@v2
       - id: mono_repo_pub_upgrade
         name: "mono_repo; pub upgrade --no-precompile"
-        run: "cd mono_repo && pub upgrade --no-precompile"
+        working-directory: mono_repo
+        run: pub upgrade --no-precompile
       - name: "mono_repo; cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate"
-        env:
-          PKGS: mono_repo
-        run: tool/ci.sh command
         if: "steps.mono_repo_pub_upgrade.conclusion == 'success'"
+        working-directory: mono_repo
+        run: "cd ../ && dart mono_repo/bin/mono_repo.dart generate --validate"
```

Comment:
Am I misremembering or did @jakemac53 add support for top level repo steps? Should this get switched?

Reply:
Not yet, sadly